tests/conftest.py

```diff
@@ -36,7 +47,42 @@ def pytest_internalerror(excinfo):
         raise excinfo.value
 
 
+def get_model_system(
+    type: str = 'original',
+    positions: List[List[float]] = [[0, 0, 0], [0.5, 0.5, 0.5]],
+    chemical_symbols: List[str] = ['Ga', 'As'],
+    orbital_symbols: List[List[str]] = [['s'], ['px', 'py']],
+) -> ModelSystem:
+    """
+    Create a `ModelSystem` object with the given parameters.
+    """
+    model_system = ModelSystem()
+    atomic_cell = AtomicCell(type=type, positions=positions * ureg.meter)
+    model_system.cell.append(atomic_cell)
+
+    # Add atoms_state to the model_system
+    atoms_state = []
+    for index, atom in enumerate(chemical_symbols):
+        orbitals = orbital_symbols[index]
+        orbitals_state = []
+        for orbital in orbitals:
+            orbitals_state.append(
+                OrbitalsState(
+                    l_quantum_symbol=orbital[0], ml_quantum_symbol=orbital[1:]
```

Comment:
This indicates that we want a setter that automatically splits (and converts) states to l and m_l.

Reply:
Yes, I have the other branch with unit testing, I can add it there and change this.